### PR TITLE
[API][Web][SIMS #906] Ministry view disbursement page

### PR DESCRIPTION
**Pull request by dheepak-aot**
- [x] On click of "View assessment" from the history table on the assessment page of a student application navigate to the assessment page
image

- [x] Display application number, institution, study date, and type, in header
image

- [x] On click of "View full assessment" display the NOA (previously from the original "view assessment" button)
- [x]  Estimated award, (same values as from the NOA)
image

- [x]  Final award (displayed from the receipt file)
image

- [x]  Enrolment confirmation, displayed based off the enrolment status for that disbursement schedule
image

image

- [x] Tuition remittance applied, displayed based off the enrolment tuition remittance for that disbursement schedule
- [x] Certificate number (This will be for Ministry Only)
- [x]  Earliest Date of Disbursement (same values as from the NOA)
- [x] if a second disbursement schedule exists, display 1-6 again
image
- [x] API created for students.

### CODE REVIEW

File: sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts [hunk 1]
```diff
@@ -172,4 +173,22 @@ export class AssessmentAESTController extends BaseController {
...
+   * @returns estimated and actual award details.
+   */
+  @Get(":assessmentId/award")
+  @ApiNotFoundResponse({
+    description: "Assessment not found.",
+  })
+  async getAssessmentAwardDetails(
+    @Param("assessmentId", ParseIntPipe) assessmentId: number,
+  ): Promise<AwardDetailsAPIOutDTO> {
+    return this.assessmentControllerService.getAssessmentAwardDetails(
```
Comment (henryoforeh-aot) [inline sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts:189]: await ?

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    <v-row>
+      <v-col>
+        <content-group>
+          <div class="category-header-medium primary-color my-3">
+            Estimated award
+          </div>
+          <!-- Estimated award table. -->
+          <award-table
+            :awardDetails="assessmentAwardData.estimatedAward"
+            identifier="disbursement1"
```
Comment (henryoforeh-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:16]: IMO a name like firstDisbursement sounds more like it.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+          />
+          <div class="my-3">
+            <status-info-label
+              :data="
+                mapCOEAssessmentProgressStatus(
+                  assessmentAwardData.estimatedAward.disbursement1Status,
```
Comment (henryoforeh-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:22]: firstDisbursementStatus.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+                )
+              "
+            />
+          </div>
+          <div
+            class="my-3"
+            v-if="
+              assessmentAwardData.estimatedAward.disbursement1TuitionRemittance
```
Comment (henryoforeh-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:30]: firstDisbursementTuitionRemittance.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+            <div>
+              <span class="label-bold">Earliest date of disbursement: </span>
+              <span>{{
+                assessmentAwardData.estimatedAward.disbursement1Date
+              }}</span>
+            </div>
+            <div v-if="isFirstDisbursementCompleted">
+              <span class="label-bold">Certificate number: </span>
+              <span>{{
+                assessmentAwardData.estimatedAward.disbursement1DocumentNumber
```
Comment (henryoforeh-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:57]: firstDisbursementDocumentNumber

File: sources/packages/web/src/assets/css/base.scss [hunk 6]
```diff
@@ -38,6 +38,7 @@ html * {
 }
 
 .content-group {
+  color: $app-content-color;
   border: 1px solid #e8e8e8;
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/base.scss:42]: Not part of the PR. Since you are working on this CSS. Pls replace #e8e8e8; with global variable

File: sources/packages/web/src/assets/css/base.scss [hunk 7]
```diff
@@ -283,3 +284,25 @@ html * {
...
   min-height: 50px;
 }
+
+.v-table.bordered {
+  border: 1px solid $common-border-color;
+}
+
+.tooltip-icon {
+  color: $color-muted;
+  font-size: 16px !important;
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/base.scss:294]: Pls use global variables. Same for line 295,304,305,306

File: sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts (hunk 1 above)
Comment (dheepak-aot) [reply sources/packages/api/src/route-controllers/assessment/assessment.aest.controller.ts:189]: Await is not required here as inside this.assessmentControllerService.getAssessmentAwardDetails there is await and it just returns values from there.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue [hunk 8]
```diff
@@ -0,0 +1,271 @@
+<template>
+  <div v-if="assessmentAwardData.estimatedAward">
+    <!-- Show the first payment/second payment header only if there is second disbursement. -->
+    <div v-if="isSecondDisbursementAvailable" class="font-weight-bold my-3">
```
Comment (ann-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: pls, use the custom class. instead of bootstrap `font-weight-bold`

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 2 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:16]: These naming conventions are in sync with the current notice of assessment way. For this API we partially need notice of assessment values which are prefixed with disbursement1 or disbursement2 and we need to re-use the logic and hence i kept the same naming convention here as well.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue [hunk 9]
```diff
@@ -0,0 +1,121 @@
+<template>
+  <v-table class="bordered">
```
Comment (ann-aot) [inline sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:2]: we have bootstrap class for bordered table https://getbootstrap.com/docs/4.1/content/tables/#bordered-table

File: sources/packages/api/src/route-controllers/assessment/assessment.controller.service.ts [hunk 10]
```diff
@@ -116,32 +129,149 @@ export class AssessmentControllerService {
...
+  /**
+   * Disbursement data is populated with dynamic key in a defined pattern to be compatible with form table.
+   * @param disbursementSchedules disbursement schedule details.
+   * @param includeDocumentNumber when true document number is mapped
+   * to disbursement dynamic data.
+   * @returns disbursement dynamic award data.
+   */
+  private populateDisbursementAwardValues(
+    disbursementSchedules: DisbursementSchedule[],
+    includeDocumentNumber?: boolean,
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/route-controllers/assessment/assessment.controller.service.ts:159]: This would be a boolean variable IMO. I would prefer to have it declared as optional with default to `false` `includeDocumentNumber = false`.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+import { PropType } from "vue";
+
+export default {
+  props: {
+    awardDetails: {
+      type: Object as PropType<DynamicAwardDTO>,
+      required: true,
+      default: {} as DynamicAwardDTO,
+    },
+    identifier: {
```
Comment (ann-aot) [inline sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:108]: for my info, what is the purpose of this prop

File: sources/packages/web/src/assets/css/base.scss (hunk 7 above)
Comment (dheepak-aot) [reply sources/packages/web/src/assets/css/base.scss:294]: Line 305,306 does not fall under a common property template to use from global variable.

image

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 8 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: why so? you meant to say I have to create a custom class which sets font weight to bold and use right?

File: sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts [hunk 12]
```diff
@@ -59,5 +59,29 @@ export class AssessmentNOAAPIOutDTO {
...
+  institutionName: string;
+  offeringIntensity: OfferingIntensity;
+  offeringStudyStartDate: string;
+  offeringStudyEndDate: string;
+  @ApiProperty({
+    description:
+      "Dynamic output of the workflow calculation. " +
+      "Contains data that could represent a part-time or a full-time award details. ",
+  })
+  estimatedAward: DynamicAwardDTO;
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts:76]: This can be declared also as `estimatedAward: Record<string, string | number>;`, if I am not wrong.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue (hunk 9 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:2]: When I apply that style to v-table it is forming a thicker border which is not looking nice and not matching figma as well.
image

File: sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts [hunk 13]
```diff
@@ -123,4 +123,49 @@ export class DisbursementReceiptService extends RecordDataModelService<Disbursem
...
+        id: true,
+        disbursementSchedule: { id: true },
+        disbursementReceiptValues: { grantType: true, grantAmount: true },
+      },
+      relations: {
+        disbursementReceiptValues: true,
+        disbursementSchedule: true,
+      },
+    };
+    if (studentId) {
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts:151]: The student id can be "appended" to the object in the same way that we would be doing with Typeorm query builder.
There is no need to repeat the code.

File: sources/packages/web/src/components/generic/DetailHeader.vue [hunk 14]
```diff
@@ -0,0 +1,25 @@
+<template>
+  <span v-for="(headerName, index) in Object.keys(headerMap)" :key="headerName">
```
Comment (ann-aot) [inline sources/packages/web/src/components/generic/DetailHeader.vue:2]: you can also use Object.entries() 
image

File: sources/packages/web/src/components/generic/DetailHeader.vue [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+
+<script lang="ts">
+import { PropType } from "vue";
+
+export default {
+  props: {
+    headerMap: {
+      type: Object as PropType<Record<string, string>>,
+      required: false,
+      default: {} as Record<string, string>,
```
Comment (ann-aot) [inline sources/packages/web/src/components/generic/DetailHeader.vue:21]: why default, if it's `required: false`?

File: sources/packages/web/src/components/generic/StatusInfoLabel.vue [hunk 16]
```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-icon :icon="icon" :color="data.status" class="status-info-icon" />
```
Comment (ann-aot) [inline sources/packages/web/src/components/generic/StatusInfoLabel.vue:2]: why `status-info-icon`??

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 8 above)
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: I believe that we have been trying to use `<p>` or `<h?>` for text to be more aligned and meaningful for accessibility.
Same for others in this file.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue (hunk 11 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:108]: This the prefix that combines with the award types to form the dynamic variable. 
e.g when identifier = disbursement1 -> disbursement1cslf, disbursement1bcag
The identifiers we assign are disbursement1, disbursement2, disbursementReceipt1, disbursementReceipt2

File: sources/packages/web/src/components/generic/TooltipIcon.vue [hunk 17]
```diff
@@ -0,0 +1,29 @@
...
+      />
+    </template>
+    <slot></slot>
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+export default {
+  props: {
+    activator: {
```
Comment (ann-aot) [inline sources/packages/web/src/components/generic/TooltipIcon.vue:17]: mm, this prop is not used in the template

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue [hunk 18]
```diff
@@ -0,0 +1,271 @@
+<template>
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:1]: Is this the component wich will be shared with the student? Should it be moved to somewhere common?

File: sources/packages/web/src/components/generic/DetailHeader.vue (hunk 14 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/DetailHeader.vue:2]: Got it. But just trying to understand what could be the benefit of using Object.entries over Object.keys

File: sources/packages/web/src/components/generic/DetailHeader.vue (hunk 15 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/DetailHeader.vue:21]: Yeah it doesn't need to be initialized. and required is true. Will change

File: sources/packages/web/src/assets/css/base.scss (hunk 7 above)
Comment (ann-aot) [reply sources/packages/web/src/assets/css/base.scss:294]: in that case, pls double-check with @hellolynn-tbtb. if it's the really font size or line size.

File: sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts (hunk 13 above)
Comment (dheepak-aot) [reply sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts:151]: I tried doing that. but couldn't achieve it in a typed fashion. May be we can sync up and see what could be done.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 8 above)
Comment (ann-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: we already have classes built for it. use that

File: sources/packages/web/src/components/generic/StatusInfoLabel.vue (hunk 16 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/StatusInfoLabel.vue:2]: To match the icon size and properties as per figma.

File: sources/packages/web/src/composables/useAssessment.ts [hunk 19]
```diff
@@ -39,5 +40,20 @@ export function useAssessment() {
...
   };
 
-  return { mapRequestAssessmentChipStatus, mapAssessmentHistoryChipStatus };
+  const mapAssessmentDetailHeader = (
+    assessment: AssessmentDetailHeader,
+  ): Record<string, string> => {
+    return {
+      "Application #": assessment.applicationNumber,
+      Institution: assessment.institutionName,
+      "Study dates": `${assessment.offeringStudyStartDate} - ${assessment.offeringStudyEndDate}`,
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/composables/useAssessment.ts:49]: I noticed that this date is coming formatted from the API. I would still prefer that the API sends the date-only format and methods like this would be adapting it to the UI format.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue [hunk 20]
```diff
@@ -0,0 +1,121 @@
...
+      <tr>
+        <th scope="col" class="text-left">Loan/grant type</th>
+        <th scope="col" class="text-left">Estimated award</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Canada Student Loan</td>
+        <td>
+          {{ awardDetails[`${identifier}cslf`] ?? NOT_ELIGIBLE }}
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:13]: This code can be changed to a method that can receive a parameter like `cslf`.

File: sources/packages/web/src/components/generic/TooltipIcon.vue (hunk 17 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/TooltipIcon.vue:17]: That property is defaulted with parent and kept for the purpose of overriding if required.

File: sources/packages/web/src/components/generic/DetailHeader.vue [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+  </span>
+</template>
+
+<script lang="ts">
+import { PropType } from "vue";
+
+export default {
+  props: {
+    headerMap: {
+      type: Object as PropType<Record<string, string>>,
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/generic/DetailHeader.vue:19]: ❤️

File: sources/packages/web/src/components/generic/DetailHeader.vue [hunk 22]
```diff
@@ -0,0 +1,25 @@
+<template>
+  <span v-for="(headerName, index) in Object.keys(headerMap)" :key="headerName">
+    <span class="label-bold-normal">{{ headerName }} : </span>
+    <span class="label-value-normal"> {{ headerMap[headerName] }} </span>
+    <!-- TODO: v-divider vertical must be used here once Vuetify 3 vertical divider is working.  -->
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/generic/DetailHeader.vue:5]: Is this TODO still needed? The `v-divider` seems to be working from Vue 3 website.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue (hunk 20 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:13]: Makes sense.

File: sources/packages/web/src/components/generic/DetailHeader.vue (hunk 22 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/DetailHeader.vue:5]: v-divider is working only as a horizontal divider. vertical divider not working(when vertical attribute is used)

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 2 above)
Comment (henryoforeh-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:16]: Sure, thanks for the explanation.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 8 above)
Comment (ann-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: image
image

File: sources/packages/web/src/components/generic/StatusInfoLabel.vue [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+</template>
+
+<script lang="ts">
+import { PropType, computed } from "vue";
+import { StatusInfo, StatusInfoDetails } from "@/types";
+
+export default {
+  props: {
+    data: {
+      type: Object as PropType<StatusInfoDetails>,
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/generic/StatusInfoLabel.vue:15]: For small component like that also are not supposed to be extended much I would prefer two properties instead of a model.

File: sources/packages/web/src/components/generic/TooltipIcon.vue (hunk 17 above)
Comment (ann-aot) [reply sources/packages/web/src/components/generic/TooltipIcon.vue:17]: I think if it's for the future. better to remove it now. and when the required come, we can adjust

**Review by ann-aot — COMMENTED** [review]
Good work @dheepak-aot  added some comments

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 8 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: I tried to find one and couldn't find one (in base.scss). Can you please help me with the class name?

File: sources/packages/web/src/services/http/dto/Assessment.dto.ts [hunk 24]
```diff
@@ -50,3 +50,26 @@ export interface AssessmentNOAAPIOutDTO {
   msfaaNumber: string;
   disbursement: any;
 }
+
+export interface DynamicAwardDTO {
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/services/http/dto/Assessment.dto.ts:54]: Can be declared using a `Record` also 😉

File: sources/packages/web/src/assets/css/base.scss (hunk 7 above)
Comment (dheepak-aot) [reply sources/packages/web/src/assets/css/base.scss:294]: Each icons are internally fonts and font-size is directly proportional to the icon size. I can check with @hellolynn-tbtb on all the properties and see if they follow a pattern to create global variables.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue (hunk 11 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:108]: I also thought the same but that will introduce changes to existing noa. 
I am just thinking is it worth the effort as this dynamic values is only for noa and this page.

File: sources/packages/web/src/views/aest/student/AssessmentAward.vue [hunk 25]
```diff
@@ -0,0 +1,90 @@
...
+    const assessmentAwardData = ref<AwardDetailsAPIOutDTO>();
+    const { mapAssessmentDetailHeader } = useAssessment();
+    const headerMap = ref<Record<string, string>>({});
+
+    onMounted(async () => {
+      assessmentAwardData.value =
+        await StudentAssessmentsService.shared.getAssessmentAwardDetails(
+          props.assessmentId,
+        );
+      headerMap.value = mapAssessmentDetailHeader(assessmentAwardData.value);
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/aest/student/AssessmentAward.vue:68]: Was `mapAssessmentDetailHeader` created as a composable because it is something that we are expecting to be shared?

**Review by andrewsignori-aot — COMMENTED** [review]
Amazing work. The UI is looking pretty good. Few comments but I guess that they are all minors.

File: sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue (hunk 8 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue:4]: you meant to say like .category-header-medium? I get it if so.

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue (hunk 11 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:108]: Just to add some points, when we create the dynamic DTO for the NOA was with the idea that the form.io could be changed to accommodate a new grant, for instance, and it would not require a code change.
Since we are moving the UI to Vuetify (I hope that NOA page will be Vuetify) maybe we can review if we still need these DTOs to be dynamic.
Apparently, even the schedule can be considered always as 1 or 2, so maybe the DTO can be more specific. Either way it does not seems to be conversation that would change this PR 😉

File: sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts (hunk 13 above)
Comment (andrewsignori-aot) [reply sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts:151]: Can you please give it a try in the below way?
image

File: sources/packages/web/src/composables/useAssessment.ts (hunk 19 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/composables/useAssessment.ts:49]: It was when we were sending the values to form.io. Still not a strong reason IMO but that was the argument provided, I guess.

File: sources/packages/web/src/views/aest/student/AssessmentAward.vue (hunk 25 above)
Comment (dheepak-aot) [reply sources/packages/web/src/views/aest/student/AssessmentAward.vue:68]: Yes. As student view will also use the header.

File: sources/packages/web/src/components/generic/DetailHeader.vue (hunk 14 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/DetailHeader.vue:2]: I see that there is a benefit of not needing to refer the object by key (obj[key]) here. I will change it.

**Review by henryoforeh-aot — APPROVED** [review]

File: sources/packages/web/src/assets/css/base.scss (hunk 7 above)
Comment (ann-aot) [reply sources/packages/web/src/assets/css/base.scss:294]: actually, if it's for the size of the icon you can use the `size` attribute of the icon. it converts to `px` if you want `vh` or `%` you can do that too

File: sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts [hunk 26]
```diff
@@ -123,4 +123,38 @@ export class DisbursementReceiptService extends RecordDataModelService<Disbursem
...
+    assessmentId: number,
+    studentId?: number,
+  ): Promise<DisbursementReceipt[]> {
+    return this.repo.find({
+      select: {
+        id: true,
+        disbursementSchedule: { id: true },
+        disbursementReceiptValues: { grantType: true, grantAmount: true },
+      },
+      relations: {
```
Comment (guru-aot) [inline sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts:146]: do we need the relations for disbursementSchedule, if we are selecting the id:true in the select? Can we have a call to check?

File: sources/packages/web/src/components/aest/students/assessment/AwardTable.vue [hunk 27]
```diff
@@ -0,0 +1,125 @@
...
+          <tooltip-icon>Canada Student Grant for Top-up.</tooltip-icon>
+        </td>
+        <td>
+          {{ getAwardValue("csgt") }}
+        </td>
+      </tr>
+      <tr>
+        <td>
+          BCAG
+          <tooltip-icon>British Colombia Access Grant.</tooltip-icon>
```
Comment (guru-aot) [inline sources/packages/web/src/components/aest/students/assessment/AwardTable.vue:65]: can we change from colombia to 'Columbia'

File: sources/packages/web/src/components/common/StatusInfoEnrollment.vue [hunk 28]
```diff
@@ -0,0 +1,50 @@
...
+  <status-info-label :status="enrollmentStatus.status">{{
+    enrollmentStatus.header
+  }}</status-info-label>
+</template>
+
+<script lang="ts">
+import { PropType, computed, defineComponent } from "vue";
+import { StatusInfo, COEStatus } from "@/types";
+
+export interface EnrollmentStatusInfo {
```
Comment (guru-aot) [inline sources/packages/web/src/components/common/StatusInfoEnrollment.vue:11]: i feel enrolment right? not enrollment.

File: sources/packages/web/src/assets/css/base.scss (hunk 7 above)
Comment (dheepak-aot) [reply sources/packages/web/src/assets/css/base.scss:294]: Yes. Just spoke to @hellolynn-tbtb and decided that for icons we are using only size attribute. font-weight does not matter.
Hence I will be removing this class status-info-icon.

File: sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts (hunk 26 above)
Comment (dheepak-aot) [reply sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts:146]: As discussed in the call, it did not work without using relations.

File: sources/packages/web/src/components/common/StatusInfoEnrollment.vue (hunk 28 above)
Comment (dheepak-aot) [reply sources/packages/web/src/components/common/StatusInfoEnrollment.vue]: enrolment (British way) and enrollment both are right. We use enrolment as confirmed by @hellolynn-tbtb

**Review by hellolynn-tbtb — APPROVED** [review]
UI looks good. We can continue to work out the acronyms, words, etc as they come up. Thanks @dheepak-aot!

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @dheepak-aot

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍

File: sources/packages/web/src/assets/css/global-style-variables.scss [hunk 29]
```diff
@@ -61,6 +61,8 @@ $badge-background-color: white;
 /** font weight bold**/
 $font-weight-bold: bold;
 
+$font-weight-bolder: bolder;
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/global-style-variables.scss:64]: why bolder? bold doesn't work??

File: sources/packages/web/src/assets/css/vuetify.scss [hunk 30]
```diff
@@ -149,3 +149,7 @@
 .v-field {
   overflow: inherit !important;
 }
+
+.v-table {
```
Comment (ann-aot) [inline sources/packages/web/src/assets/css/vuetify.scss:153]: IMO if it is only for a one-page v-table. better to custom class specific to the page. if it's the same across the project. then it's good. but I think v-table can be used for different purposes,

File: sources/packages/web/src/assets/css/global-style-variables.scss (hunk 29 above)
Comment (dheepak-aot) [reply sources/packages/web/src/assets/css/global-style-variables.scss:64]: It was created for icons. Will remove it as per my previous comment.

File: sources/packages/web/src/assets/css/vuetify.scss (hunk 30 above)
Comment (dheepak-aot) [reply sources/packages/web/src/assets/css/vuetify.scss:153]: It is same across the object. Otherwise it is setting font color to 00000 (black)

File: sources/packages/web/src/components/common/AssessmentAwardDetails.vue [hunk 31]
```diff
@@ -0,0 +1,233 @@
+<template>
+  <div v-if="assessmentAwardData.estimatedAward">
+    <!-- Show the first payment/second payment header only if there is second disbursement. -->
+    <h3
+      v-if="isSecondDisbursementAvailable"
+      class="category-header-medium secondary-color my-3"
```
Comment (ann-aot) [inline sources/packages/web/src/components/common/AssessmentAwardDetails.vue:6]: mm, is it not brand black?? 
image

File: sources/packages/web/src/components/common/AssessmentAwardDetails.vue [hunk 32]
```diff
@@ -0,0 +1,233 @@
...
+              )
+            }}
+          </div>
+        </content-group>
+      </v-col>
+    </v-row>
+  </div>
+  <!-- Estimated and actual award details of second disbursement. -->
+  <div v-if="isSecondDisbursementAvailable">
+    <h3 class="category-header-medium secondary-color my-3">Second Payment</h3>
```
Comment (ann-aot) [inline sources/packages/web/src/components/common/AssessmentAwardDetails.vue:86]: mm. same here. can you double check

File: sources/packages/web/src/components/generic/TooltipIcon.vue [hunk 33]
```diff
@@ -0,0 +1,26 @@
...
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+export default {
+  props: {
+    location: {
+      type: String,
+      required: false,
+      default: "right",
```
Comment (ann-aot) [inline sources/packages/web/src/components/generic/TooltipIcon.vue:22]: mm, I think agreed not to set aby default location right?

Comment (dheepak-aot) [reply sources/packages/web/src/components/generic/TooltipIcon.vue:22]: We tested it and even by setting default location the responsiveness was not affected and hence the default location property was restored.

**Review by ann-aot — APPROVED** [review]
Awsome work @dheepak-aot 👍 Thanks for doing the changes
